iontestdata_1_0/good/non-equivs/symbolTablesUnknownText.ion

```diff
@@ -1,25 +1,50 @@
-// import declaration, with max_id > 0
+// verifies that imported symbols with undefined text are not equivalent to:
+// - other imported symbols with undefined text with different SIds
+// - local symbols with undefined text
+// - symbols with defined text
 embedded_documents::[
     '''
     $ion_1_0
     $ion_symbol_table::{
-        symbols:[ "foo", "bar", "baz" ]
+        imports:[ { name: "com.amazon.ion.tests",
+                    version: 1,
+                    max_id: 2 }
+        ],
+        symbols:[ "foo", 31 ]
```

Comment:
From https://amazon-ion.github.io/ion-docs/docs/symbols.html#local-symbol-tables 

> The `symbols` field should be a list of strings. If the field is missing or has any other type, it is treated as if it were an empty list.
> 
> Null elements in the symbols list declare unknown symbol text (“gaps”) for its SID within the sequence. Any element of the list that is not a string must be interpreted as if it were null. Any SIDs that refer to null slots in a local symbol table are equivalent to symbol zero.

What is this doing, why put a number here instead of an explicit `null`? Why 31?

Reply:
This could be any non-string (or null), testing the case where"Any element of the list that is not a string must be interpreted as if it were null."

I'm sure we have tests elsewhere that ensure the implementation treats all non-strings as null slots.